test/tools/isoltest.cpp

```diff
@@ -287,7 +287,7 @@ TestStats TestTool::processPath(
 				_testCaseCreator,
 				_options,
 				fullpath,
-				currentPath.string()
+				currentPath.generic_path().string()
```

Comment:
I only find that maked as `experimental`. What about `generic_string`?

Reply:
I was aware of that, but the comment was saying more than just "experimental": why they think this function is better than its family of alternatives, that's why I decided for this one.

OTOH, when we someone is picking up the task to move from boost::filesystem to std::filesystem, we can adapt that (?).